kim/kim-ldap/src/main/java/org/kuali/rice/kim/dao/impl/LdapPrincipalDaoImpl.java

```diff
@@ -57,6 +59,7 @@
  */
 public class LdapPrincipalDaoImpl implements LdapPrincipalDao {
     protected Constants kimConstants;
+    protected UaEdsConstants edsConstants;
```

Comment:
I didn't see UaEdsConstants in this pull request. I assume it's already commited in another ticket?